src/gui/MainWindow.cpp

```diff
@@ -308,12 +308,13 @@ void MainWindow::finalize()
 					SLOT( exportProjectTracks() ),
 					Qt::CTRL + Qt::SHIFT + Qt::Key_E );
 
-	// temporarily disabled broken MIDI export				
-	/*project_menu->addAction( embed::getIconPixmap( "midi_file" ),
+#ifdef LMMS_HAVE_FLUIDSYNTH
```

Comment:
Well, you can change them as you wish. 

In my opinion, LMMS should have ablity to output a pretty MIDI file, for I had thousands of MIDI file to listen.